py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -138,7 +141,10 @@ def _compile_module(
             settings=settings,
         )
 
-        # Replace FX Module with TRT Module
+        trt_modules[name] = trt_mod
+
+    # Replace all FX Modules with TRT Modules
+    for name, trt_mod in trt_modules.items():
         setattr(partitioned_module, name, trt_mod)
```

Comment:
Set all TRT engine attributes _after_ completion of compilation. Forward passes during compilation are run on the Torch equivalents, _not_ the TRT graphs.